modules/nf-core/gatk4/filterintervals/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: "gatk4_filterintervals"
+description: Filters intervals based on annotations and/or count statistics.
+keywords:
+  - gatk4
+  - gatk4_filterintervals
+tools:
+  - "gatk4":
+      description:
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/categories/360002369672s
+      doi: "10.1158/1538-7445.AM2017-3590"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - tsv:
```

Comment:
This should be intervals/input depending on what we go with in the main.nf file

Reply:
I've changed it to input. The annotated_intervals file is a tsv file. Should I change the input path from "annotated_intervals" to "tsv" rather?